### PR TITLE
Add chicken hero

### PR DESCRIPTION
**Pull request by darabos**
A review would be nice while I fix the conflict. Thanks!

### CODE REVIEW

File: backend/actions.py [hunk 1]
```diff
@@ -147,18 +146,33 @@ def hankering(self):
...
+    if target:
+      target.status.append({'type': 'SafetyCollar', 'damage': self.subject.influence * 5})
+
+
 class ComeToPapa(Action):
   default_hankering = 10
   pull_range = 1
   cooldown = 5
-  inspiration = 1
+  inspiration = 3
```
Comment (darabos) [inline backend/actions.py:175]: If there's an inspiration=1 action, the user barely even sees the inspiration. We get it and immediately spend it. I think it's more interesting to require 3 or 5 as the default.

File: backend/heroes.py [hunk 2]
```diff
@@ -210,12 +254,14 @@ class CrocodileMan(Hero):
...
-  def before_step(self):
-    normal_influance = self.influence_base + self.level * self.influence_per_level
-    self.influence = normal_influance * (
-      (self.max_loyalty - abs(self.loyalty)) / self.max_loyalty + 1)
+  anger = 0
+  def hit(self, amount):
+    super().hit(amount)
+    self.anger += amount
+    normal_influence = self.influence_base + self.level * self.influence_per_level
+    self.influence = normal_influence * (self.anger / self.max_loyalty + 1)
```
Comment (darabos) [inline backend/heroes.py:264]: I changed it so that he doesn't get weaker from healing.

File: backend/actions.py [hunk 3]
```diff
@@ -147,18 +146,33 @@ def hankering(self):
...
+
+class SafetyCollar(Action):
+  inspiration = 3
+  def hankering(self):
+    return 4 if self.target else 0
+  def prepare(self, state):
+    self.target = self.subject.find_closest_opponent(state)
+  def apply_effect(self):
+    if target:
+      target.status.append({'type': 'SafetyCollar', 'damage': self.subject.influence * 5})
```
Comment (xandrew) [inline backend/actions.py:168]: I'm probably just missing this, but where is status initialized?
I don't mind too much, but why not just a safety_collar instance variable with the total amount of collar damage? I mean why this general status machinery, do you plan stuff like clear all status?

File: backend/heroes.py (hunk 2 above)
Comment (xandrew) [reply backend/heroes.py:264]: Looks good!

**Review by xandrew — COMMENTED** [review]
You scared me in the chat that you did something terrible, so I had to take a look! :) But LGTM!

File: backend/actions.py (hunk 3 above)
Comment (darabos) [reply backend/actions.py:168]: > I'm probably just missing this, but where is status initialized?

In the constructor. I didn't add it. It was already there.

> I don't mind too much, but why not just a safety_collar instance variable with the total amount of collar damage? I mean why this general status machinery, do you plan stuff like clear all status?

I think the status stuff is also sent to the frontend, so we can graphically indicate this stuff. I like it. I'm doing the mushrooms with this too.

I didn't think adding random instance variables to Hero was fair play. But it's great if it is! 😄

**Pull request merged**